controller/backup_target_controller.go

```diff
@@ -337,6 +357,47 @@ func (btc *BackupTargetController) reconcile(name string) (err error) {
 		}
 	}
 
+	// start a BackupStoreTimer and keep it in a map[string]*BackupStoreTimer
+	stopTimer := func(backupTargetName string) {
+		_, exists := btc.bsTimerMap[backupTargetName]
+		if exists {
+			btc.bsTimerMap[backupTargetName].Stop()
+			delete(btc.bsTimerMap, backupTargetName)
+		}
```

Comment:
`btc.bsTimerMap` is initialized when the backup target controller is created. It should never be `nil`, so can you point out where it will be set `nil`?